posts/2022/10-01-api_queries_with_finch.md

```diff
@@ -0,0 +1,173 @@
+==title==
+API queries with Finch
+
+==author==
+Cornelia Kelinske
+
+==description==
+I've been using HTTPoison as my HTTP client of choice ever since I learned how to do API queries with Elixir.
+Lately, though, I've been hearing more about people using Finch for that purpose. So I went ahead and tried it out.
+
+
+==tags==
+coding, elixir, api_calls, finch
+
+==body==
+
+# 1. The set-up
+
+The first thing that is good to know about Finch is that we want to add it to our supervision tree. This means that, ideally, we'll remember to create our new app with the `mix new new_app --sup` command so that we get our `application.ex` file with the supervision tree automatically generated.
```

Comment:
```suggestion
The first thing that is good to know about Finch is that we want to add it to our supervision tree. Generate a new project with `mix new new_app --sup` command so that we get our `application.ex` file with the supervision tree automatically generated.
```